lang/test/org/partiql/lang/syntax/SqlParserDateTimeTests.kt

```diff
@@ -8,14 +8,12 @@ import org.partiql.lang.domains.id
 import java.util.*
 import org.partiql.lang.errors.ErrorCode
 import org.partiql.lang.errors.Property
-import org.partiql.lang.util.softAssert
+import org.partiql.lang.util.LOCAL_TIMEZONE_OFFSET
```

Comment:
I might have missed this, but what is it--is it a statically defined local timezone offset based on JVM settings?

Reply:
It is, yes. But this will be changed to `+00:00` as discussed in a separate PR.